test/scenarios/attestation/output/src/attestation/azext_attestation/tests/latest/example_steps.py

```diff
@@ -12,76 +12,25 @@
 from .. import try_manual
 
 
-# EXAMPLE: /AttestationProviders/get/AttestationProviders_Get
+# EXAMPLE: AttestationProviders_Create
 @try_manual
-def step_attestation_provider_show(test, rg, rg_2, rg_3, checks=None):
-    if checks is None:
-        checks = []
-    test.cmd('az attestation attestation-provider show '
-             '--provider-name "myattestationprovider" '
-             '--resource-group "{rg}"',
-             checks=checks)
-
-
-# EXAMPLE: /AttestationProviders/get/AttestationProviders_List
-@try_manual
-def step_attestation_provider_provider_list(test, rg, rg_2, rg_3, checks=None):
-    if checks is None:
-        checks = []
-    test.cmd('az attestation attestation-provider provider list '
-             '-g ""',
-             checks=checks)
-
-
-# EXAMPLE: /AttestationProviders/get/AttestationProviders_ListByResourceGroup
-@try_manual
-def step_attestation_provider_provider_list2(test, rg, rg_2, rg_3, checks=None):
+def step_create_provider(test, rg, rg_2, rg_3, checks=None):
     if checks is None:
         checks = []
-    test.cmd('az attestation attestation-provider provider list '
-             '--resource-group "{rg_2}"',
+    test.cmd('az attestation create-provider '
+             '--resource-group "{rg_4}" '
+             '--provider-name "myattestationprovider"',
```

Comment:
Also, Can we remove the unused rg* ?

Reply:
rg* is designed for manual override. 
Since it's redundant in most case, two features are design for remove this:
1. add a flag to use unique rg (already in this PR) https://github.com/Azure/autorest.az/pull/736/files/657e4f88a21750f1d4117460a468c0e8b41ebd1f#diff-c5435eb3f58e359805502547569b00b540007366f273d01647052b0e35d385bdR211
2. remove rg* in function parameter list (in coming PR)